apps/liquidation-bot/src/metrics/dd_client.ts

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+
+const DD_BASE_URL = "https://api.datadoghq.com";
+
+export type DDEvent = {
+  key: string | null;
+  type: string;
+  text: string;
+  title: string;
+  tags: string[];
+  timestamp: number;
+};
+
+export type DDMetric = {
+  name: string;
+  value: number;
+  tags: string[];
+  timestamp: number;
+};
+
+export default class DDClient {
```

Comment:
There is the `@notional-finance/logging` package now, not sure if you can move some of this code to use that library instead?